build.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh
+
+set -eu
+
+ARCH="${ARCH:-armhf}"
+CUR_DIR="$(pwd)"
+BUILD_DIR="${CUR_DIR}/.build_${ARCH}"
+DEBIAN_VERSION="${DEBIAN_VERSION:-jessie}"
+ROOTFS_ARCHIVE="rootfs.tar.xz"
+ROOTFS_DIR="${BUILD_DIR}/rootfs"
+QEMU_STATIC_BIN="$(command -v qemu-arm-static)"
+
+
+cleanup()
+{
+    local mount_points
+    mount_points="$(mount | grep "${ROOTFS_DIR}")"
```

Comment:
a small optimization is to just grep in /proc/mounts, as mount just prints what /proc/mounts says.